Users/templates/Users/profile.html

```diff
@@ -1,12 +1,113 @@
-{% extends "Character_Builder/Character_base.html" %}
-{% block content %}
-	<div class="content-section">
-		<div class="media">
-			<img class="rounded-circle account-img" src="{{ user.profile.image.url }}">
-			<div class="media-body">
-				<h2 class="account-heading">{{ user.username }}</h2>
-				<p class="text-secondary">{{ user.email }}</p>
+{% extends "Character_Builder/base.html" %}
+{% block main %}
+
+<main role="main" class="container">
+	<div class="row">
+		<div class="col-md-12">
+			<div class="content-section">
+				<div class="media">
+					<img class="rounded-circle account-img" src="{{ user.profile.image.url }}">
+					<div class="media-body">
+						<h2 class="account-heading">{{ user.username }}</h2>
+						<p class="text-secondary">{{ user.email }}</p>
+					</div>
+				</div>
 			</div>
+
+<!--
+			<div class="content-section">
+				<div class="media">
+					<div class="media-body">
+						<h2 class="account-heading">Current Campaigns</h2>
+
+						{% for Campaign in campaigns %}
+							<div class="media">
+								<div class="media-body">
+									<h2>{{ Campaign.campaignName }}</h2>
+									<h3>Campaign ID: {{ Campaign.campaignID }}</h3>
+								</div>
+							</div>
+						{% endfor %}
+
+					</div>
+				</div>
+			</div>
+-->
```

Comment:
Do you need this? If not, just delete this whole commented section

Reply:
Thanks for pointing this out. I forgot to remove it after testing.